apps/hash-graph/lib/graph/src/store/test_graph.rs

```diff
@@ -133,10 +133,27 @@ impl CustomGlobalMetadata {
     }
 }
 
+#[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct BlockProtocolModuleVersions {
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub graph: Option<semver::Version>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub authorization: Option<semver::Version>,
```

Comment:
Sorry this doesn't exist, it was just an example of what we might want to do in the future 😅 

I'm unclear if `graph` should be optional, I think let's make it not optional for now, we can always change it in a compatible way later on